### ISSUE TITLE
Random crash when exploring Nordic Europa

### ISSUE THREAD

**Issue by MyDudeYo** [NomadsReach/Fallout-Anomaly#79]
[crash-2024-04-04-21-50-13.log](https://github.com/NomadsReach/Fallout-Anomaly/files/14877672/crash-2024-04-04-21-50-13.log)
Please use the below as a template when making your report.

## Describe the bug
I just clearing the building of ghouls, and when I find the card key to open the sliding door where a bunch of ghouls are, game will crash.

### Is it already a known issue?
If so or you are not sure please do not make a new request and see the known issue page. [Here](https://github.com/NomadsReach/Fallout-Anomaly/blob/master/Known%20Issue.md)

### Steps to reproduce
clearing out ghouls in Nordic Europa building. 

### Environment
 - OS: Windows 11 Home. 
 - Hardware ; CPU : 7800X3D, GPU ROG 4090 OC, RAM G.Skills DDR5 6400. 
 - Other details that you think may affect.
   

### Contact: 
    
   Please list your discord name if you are in our discord: my_dude_yo
[crash-2024-04-04-21-40-59.log](https://github.com/NomadsReach/Fallout-Anomaly/files/14877691/crash-2024-04-04-21-40-59.log)
[crash-2024-04-04-21-50-13.log](https://github.com/NomadsReach/Fallout-Anomaly/files/14877693/crash-2024-04-04-21-50-13.log)

---

### Additional context
Add any other context about the problem here or screenshots/videos/ticket transcript from discord.

---
[crash-2024-04-04-21-40-59.log](https://github.com/NomadsReach/Fallout-Anomaly/files/14877698/crash-2024-04-04-21-40-59.log)

**Comment by NomadsReach** [comment]
Please upload your save file.

**Comment by MyDudeYo** [comment]
Hmm. when I drop the save file in to upload, I get a "We don't support that file type". The file type is F4SE file.

**Comment by MyDudeYo** [comment]
[Save36_F31E80A3S5354414C4B4552_Cmbfacility02_000530_20240404231902_21_2.zip](https://github.com/NomadsReach/Fallout-Anomaly/files/14878218/Save36_F31E80A3S5354414C4B4552_Cmbfacility02_000530_20240404231902_21_2.zip)

**Comment by MyDudeYo** [comment]
He Nomad, the actual building I am have the issue in is the Decrepit Building (I am not too sure why the save is saying the Nordic Europa?) just next door to the Cambridge Police Station.  I loaded up a previous save right before I entered the building and have been avoiding that place for now, because of the crash.  

FYI, I uploaded my save file. I needed to send to a zip file. Hopefully that works.  again, thanks for the help!
